lib/ansible/modules/systemd.py

```diff
@@ -357,8 +404,30 @@ def main():
 
     systemctl = module.get_bin_path('systemctl', True)
 
-    if os.getenv('XDG_RUNTIME_DIR') is None:
-        os.environ['XDG_RUNTIME_DIR'] = '/run/user/%s' % os.geteuid()
+    xdg = os.getenv('XDG_RUNTIME_DIR')
+    xdg_path = '/run/user/%s' % os.geteuid()
```

Comment:
This probably needs to be bytes.

Reply:
the user name one does, this one is always an int .. but i'll make em both bite anyways